### PR TITLE
FORM-243: Form header - visual

### PR DESCRIPTION
**Pull request by AntonZezul**

### CODE REVIEW

File: next/components/forms/icon-components/Waves.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
+type WaveProps = {
+  className?: string
+  waveColor?: string
+  innerLinesColor?: string
+  outerLinesColor?: string
+  isRich?: boolean
+}
+
+const Waves = ({
```
Comment (radoslavzeman) [inline next/components/forms/icon-components/Waves.tsx:9]: Please get the newer version of Wawes from bratislava.sk project. This file is too old an doutdated.

File: next/components/forms/simple-components/MenuDropdown/MenuTrigger.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          'text-gray-800 border-gray-800': isOpen && buttonVariant === 'gray',
+        },
+        {
+          'w-10 h-10 rounded-lg border-2 border-main-700 bg-transparent text-gray-700 focus:border-main-800 focus:text-gray-800':
+            buttonVariant === 'main',
+          'text-gray-800 border-main-800': isOpen && buttonVariant === 'main',
+        },
+        {
+          '': buttonVariant === 'none',
+          d: isOpen && buttonVariant === 'none',
```
Comment (radoslavzeman) [inline next/components/forms/simple-components/MenuDropdown/MenuTrigger.tsx:34]: What is `d` class?

File: next/components/forms/simple-components/AriaMenuComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// const MyItem = (props: ItemProps) => {
+//   return (
+//     <Item
+//       {...props}
+//       className={({ isFocused, isSelected }) => `my-item ${isFocused ? 'focused' : ''}`}
+//     />
+//   )
+// }
+
+const AriaMenuComponent = <T extends object>({
```
Comment (radoslavzeman) [inline next/components/forms/simple-components/AriaMenuComponent.tsx:16]: Do you use this component? If not, please remove and uninstall `react-aria-components` lib, if not used.

File: next/components/forms/simple-components/MenuDropdown/MenuTrigger.tsx (hunk 2 above)
Comment (AntonZezul) [reply next/components/forms/simple-components/MenuDropdown/MenuTrigger.tsx:34]: my bad, it is just need to remove

File: next/components/forms/segments/AccountNavBar/AccountNavBar.tsx [hunk 4]
```diff
@@ -17,6 +17,7 @@ import NextLink from 'next/link'
 import { useRouter } from 'next/router'
 import { useTranslation } from 'next-i18next'
 import { ReactNode, useState } from 'react'
+import { RemoveScroll } from 'react-remove-scroll'
```
[the remark this answers is not in the document]
Comment (AntonZezul) [reply next/components/forms/segments/AccountNavBar/AccountNavBar.tsx:21]: We decided to keep used RemoveScroll

**Pull request merged**